src/resonanceReconstruction/rmatrix/src/fromENDF.hpp

```diff
@@ -1,7 +1,14 @@
-inline Reconstructor
+Reconstructor
 fromENDF( const ENDF::ResonanceRange& endfResonanceRange,
           const AtomicMass& neutronMass,
-          const ElectricalCharge& elementaryCharge ) {
+          const ElectricalCharge& elementaryCharge,
+          const ParticleID& incident,
+          const ParticleID& target ) {
+
+  auto lower = endfResonanceRange.lowerEnergy();
+  auto upper = endfResonanceRange.upperEnergy();
+  auto nro = endfResonanceRange.energyDependentScatteringRadius();
+  auto naps = endfResonanceRange.scatteringRadiusCalculationOption();
```

Comment:
Should these be `const`?
```suggestion
const auto lower = endfResonanceRange.lowerEnergy();
const auto upper = endfResonanceRange.upperEnergy();
const auto nro = endfResonanceRange.energyDependentScatteringRadius();
const auto naps = endfResonanceRange.scatteringRadiusCalculationOption();
```

Reply:
We can do that.